app/api/gql/gql.ts

```diff
@@ -43,6 +45,12 @@ export function graphql(source: string): unknown;
 export function graphql(
   source: "query FetchExampleEvents {\n  events {\n    id\n    description\n    community {\n      id\n      name\n    }\n    tags {\n      id\n      name\n      description\n    }\n  }\n}",
 ): (typeof documents)["query FetchExampleEvents {\n  events {\n    id\n    description\n    community {\n      id\n      name\n    }\n    tags {\n      id\n      name\n      description\n    }\n  }\n}"];
+/**
+ * The graphql function is used to parse GraphQL queries into a document that can be used by GraphQL clients.
+ */
+export function graphql(
+  source: "query misTickets($input: PaginatedInputMyTicketsSearchValues!) {\n  myTickets(input: $input) {\n    data {\n      approvalStatus\n      id\n      paymentStatus\n      redemptionStatus\n    }\n    pagination {\n      currentPage\n      pageSize\n      totalPages\n      totalRecords\n    }\n  }\n}",
```

Comment:
are we going to name operations in english or spanish? we should choose one and follow that pattern going forward.

Reply:
Agreed. English :)

Reply:
Will do a pass in the API to update naming